File: app/src/main/java/com/pkfcooperparry/template/ui/form/page/tab/TabFragmentProvider.java
package com.pkfcooperparry.template.ui.form.page.tab;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class TabFragmentProvider {

    @ContributesAndroidInjector(modules = TabFragmentModule.class)
    abstract TabFragment provideStatsFragmentFactory();
}

File: app/src/main/java/com/pkfcooperparry/template/ui/form/page/tab/TabFragmentProvider.kt
package com.pkfcooperparry.template.ui.form.page.tab

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class TabFragmentProvider {

    @ContributesAndroidInjector(modules = [TabFragmentModule::class])
    internal abstract fun provideStatsFragmentFactory(): TabFragment
}
